apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -27,6 +27,8 @@ import { Card } from '@/shared/components/card';
 import { Route } from '@/shared/constants';
 
 type CompanyReviewProps = {
+  id?: string;
+  date: string;
```

Comment:
It looks like `date` is already present in line 37 of this file so it is not needed here